profile/store_test.go

```diff
@@ -45,7 +46,12 @@ func TestPutProfileWithAddresses(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	ps, err := NewLocalStore(filepath.Join(path, "profiles.json"), &Profile{PrivKey: kd0.PrivKey, Peername: "user"}, ks)
+	owner := &Profile{
+		ID:       ID(kd0.PeerID),
```

Comment:
As per the description of https://github.com/qri-io/qri/pull/1121, don't call `profile.ID` directly. Instead, use `IDFromPeerID`.

Reply:
lovely. There are a bunch of direct `ID() type-coercions in this file (looks like #1121 grepped for only calls _outside_ the  profile package) will fix.